app/src/main/java/de/upb/hip/mobile/activities/ExhibitDetailsActivity.java

```diff
@@ -624,14 +614,18 @@ private void updatePlayPauseButtonIcon() {
 
     }
 
-    /**
-     * Modifies the visibility of the audio action in the toolbar.
-     *
-     * @param visible True indicates the audio action should be visible.
-     */
-    private void displayAudioAction(boolean visible) {
```

Comment:
A lot of code is removed in the ExhibitDetailsActivity which was added to make the audio action in the toolbar visible on demand. That code should not be removed. This seems to have occurred during merging.